### ISSUE TITLE
_prepare_path crashes if passed pathlib.Path 

### ISSUE THREAD

**Issue by mycroftw** [szmikler/playsound3#5]
According to README and docstring, playsound() accepts "Path or URL to the sound file. Can be a string or pathlib.Path."

However, if a `pathlib.Path` is provided, when it is sent to `_prepare_path`, it errors out with:
```
  File "[Path removed]\validators.py", line 253, in _validate
    playsound(Path("../sound/default.mp3"))
  File "[Path removed]\venv\Lib\site-packages\playsound3\playsound3.py", line 44, in playsound
    path = _prepare_path(sound)
           ^^^^^^^^^^^^^^^^^^^^
  File "[Path removed]\venv\Lib\site-packages\playsound3\playsound3.py", line 61, in _prepare_path
    if sound.startswith(("http://", "https://")):
       ^^^^^^^^^^^^^^^^
AttributeError: 'WindowsPath' object has no attribute 'startswith'
python-BaseException
```
Python 3.12, windows 11, but I think it's a global issue.

**Comment by szmikler** [comment]
Thanks for the report and sorry for the late response. Fixed in 2.2.3.

**Comment by mycroftw** [comment]
Tested, works as documented.  Thanks!
